packages/block-editor/src/components/media-placeholder/README.md

```diff
@@ -133,9 +133,9 @@ An object that can contain a `title` and `instructions` properties. These proper
 
 ### multiple
 
-Whether to allow multiple selection of files or not.
+Whether to allow multiple selection of files or not. This property will also accept a string with the value `add` to allow multiple selection of files without the need to use the `Shift` or `Ctrl`/`Cmd` keys.
 
--   Type: `Boolean`
+-   Type: `Boolean|String`
```

Comment:
Maybe the `MediaPlaceholder` docs should only be updated once the types have been changed here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/wordpress__block-editor/components/media-placeholder.d.ts

Reply:
Done in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66596 🎉